### DISCUSSION TITLE
Is is possible to disable a particular major version update from being re-created?

### DISCUSSION THREAD

**Discussion by cy6erskunk** [renovatebot/renovate#10551] [Request Help] [answered]
We would like to skip a major version for a particular package, is it possible to configure renovate to stop creating PRs for a given version?

**Comment by viceice** [comment, marked as answer]
Yes, use dependency dashboard approval or `allowedVersions`

**Reply by cy6erskunk** [reply]
Dependency dashboard approval is a bit of overkill, but `allowedVersions` should work. It's not quite clear from the docs what's the correct syntax for multiple rules, would it be `<5.0.0 || >=6.0.0` or `<5.0.0 or >=6.0.0`  in case we would like to skip version `5`?

**Reply by rarkins** [reply]
It must be a valid constraint for the language. e.g. if it's npm then it needs to be valid npm semver like you'd put in `package.json`

**Reply by cy6erskunk** [reply]
Thanks a lot, according to [sevmver docs](https://github.com/npm/node-semver#versions) `<5.0.0 || >=6.0.0` is what I need
